src/content/getting-started/observe-your-clusters-and-apps/index.md

```diff
@@ -69,42 +69,21 @@ Along with service monitor metrics, you can check metrics for DNS, Ingress, Flux
 
 ## Step 3: Create your own dashboard
 
-In some occasions, you may want to create a custom dashboard to visualize the metrics of your application. Once logged into the [platform API]({{< relref "/reference/platform-api" >}}) you can create a special `ConfigMap` containing the dashboard in JSON format. It will look like this:
+In some occasions, you may want to create a custom dashboard to visualize the metrics of your application. Once logged into the [platform API]({{< relref "/reference/platform-api" >}}) you can start using the Grafana UI and create your very first dashboard. There, you can create panels with the desired visualizations and then export your dashboard by selecting `Share > Export`` in the dashboard context menu or by accessing the JSON Model in the dashboard settings.
```

Comment:
Another thing here...do you really need to be logged in to the platform-api just to use Grafana? This sounds like a bit of legacy, doesn't it? You can just open Grafana if you have the link (or use backstage to get there :D)

Reply:
Oh and why do you showcase the export here? Since we're not showing the ConfigMap approach we don't really need to export a dashboard, right?